src/main/webapp/i18n/de/lectureUnit.json

```diff
@@ -130,7 +130,7 @@
                 "version": "Version",
                 "fileRequiredValidationError": "Du musst eine Datei zum uploaden auswählen.",
                 "chooseFile": "Datei auswählen",
-                "fileLimitations": "Unterstützte Formate: \".png\", \".jpg\", \".jpeg\", \".svg\", \".pdf\", \".zip\". Max. Dateigröße 10 mb"
+                "fileLimitations": "Unterstützte Formate: \".png\", \".jpg\", \".jpeg\", \".svg\", \".pdf\", \".zip\", \".doc\", \".docx\", \".csv\", \".xls\", \".xlsx\", \".ppt\", \".pptx\", \".pages\", \".rtf\", \".numbers\", \".key\", \".odt\", \".ods\", \".odp\", \".odg\", \".odf\". Max. Dateigröße 10 mb"
```

Comment:
Looks a bit weird in my opinion. Maybe just write “document formats” and/or add a tooltip that reveals the individual file extensions?

<img width="500" alt="Screenshot 2022-07-05 at 09 34 28" src="https://user-images.githubusercontent.com/102237363/177275146-e72d1c46-a648-4ce7-9bc7-7ae24adb4753.png">